cpp/order/LocalWl.cc

```diff
@@ -79,7 +79,7 @@ void LocalWl::compute(const locality::NeighborList* nlist, const vec3<float>* po
             {
                 const unsigned int index = (2 * m_l + 1) * i;
                 const unsigned int u3 = 3 * m_l - u1 - u2;
-                m_Wli.get()[i] += m_wigner3jvalues[counter] * m_Qlmi.get()[index + u1]
+                m_Wli.get()[i] += float(m_wigner3jvalues[counter]) * m_Qlmi.get()[index + u1]
```

Comment:
Why are all these float conversions necessary? The object is a std::vector, it should already return floats.

Reply:
OK, I see that you changed that too. Why? Were you getting compiler warnings from hardcoding numbers in the wigner3j.cc file that overflow float precision?

Reply:
Yeah, I was getting warnings. I changed the return type to `vector<float>` and it was interpreting the vector literals `{0.0, 0.1, ...}` as doubles. This was the easier thing to change.

Reply:
What caused errors that didn't cause errors before? Was it because you increased the precision of the numbers hard-coded in the file?